posthog/queries/person_query.py

```diff
@@ -81,10 +91,20 @@ def get_query(self, prepend: str = "") -> Tuple[str, Dict]:
             {cohort_query}
             WHERE team_id = %(team_id)s
             GROUP BY id
-            HAVING max(is_deleted) = 0 {person_filters} {search_clause}
+            HAVING max(is_deleted) = 0
+            {person_filters} {search_clause} {distinct_id_clause} {email_clause}
+            {"ORDER BY max(created_at) DESC, id" if paginate else ""}
```

Comment:
any specific reason for choosing this? 🤔 , doesn't seem to add anything, since each `id` will only have one row?

Reply:
It's an order by, not a group by. The problem is we may well have multiple people created at the exact same time, which messes up the ordering. Adding id makes sure the ordering is consistent. You can verify this by running `TestPerson.test_pagination_limit` and removing `id`

Reply:
Sorry, I meant the `max(created_at) DESC`, not the id.

But the same reason you say makes sense!